src/main/java/org/fcrepo/importexport/exporter/Exporter.java

```diff
@@ -289,14 +302,23 @@ private void exportDescription(final URI uri) throws FcrepoOperationFailedExcept
             exportLogger.error(String.format("Error exporting description: {}, Cause: {}", uri, ex.getMessage()), ex);
         }
 
-        exportMembers(file);
+        exportMembers(file, uri);
     }
```

Comment:
This is a minor stylistic point, and if you have a specific personal preference for omitting whitespace after the `}` at the end of this method, I would be interested to hear it.
Otherwise, for legibility, please add a newline between these two methods.